usr/share/rear/prep/NBKDC/default/40_prep_nbkdc.sh

```diff
@@ -0,0 +1,93 @@
+#
+# prepare stuff for NovaBACKUP DataCenter
+#
+
+# detect where and if DataCenter client is installed on the system running
+# REAR and include the needed bits in the recovery ISO
+
+if [ ! -f "$NBKDC_DIR/conf/client.properties" ]; then
+    NBKDC_DIR=$(readlink -f /proc/$(pgrep -nx rcmd-executor)/exe | sed 's/\/rcmd-executor*//g')
+    if  [ ! -f "$NBKDC_DIR/conf/client.properties" ] ; then
+        echo "Cannot find running NovaBACKUP DataCenter Agent!!"
+        echo "Locating the NovaBACKUP DataCenter Agent via /Hiback"
+        NBKDC_DIR=$(readlink /Hiback | sed 's/\/Hiback$//g')
+        if [ ! -f "$NBKDC_DIR/conf/client.properties" ]; then 
+            echo "No NovaBACKUP DataCenter Software installed"
+            Log "No NBKDC found, exiting NBKDC prep"
+            exit 2
+        fi
+    fi
+fi
```

Comment:
And what happens if `"$NBKDC_DIR/conf/client.properties"` does not exist? Nothing? Will it still work?